packages/esm-patient-medications-app/src/order-basket/order-basket-item-list.scss

```diff
@@ -1,7 +1,70 @@
 @import "../root";
 
+.actionText {
+  @include carbon--type-style("body-short-01");
+  color: $text-02;
+  margin-bottom: 1rem;
+}
+
+.content {
+  @include carbon--type-style("productive-heading-01");
+  color: $text-02;
+  margin-top: $spacing-05;
+  margin-bottom: $spacing-03;
+}
+
+.heading {
+  text-align: left;
+  text-transform: capitalize;
+  margin-bottom: 1rem;
+  @include carbon--type-style("productive-heading-03");
+  color: $text-02;
+}
+
+.heading:after {
+  content: "";
+  display: block;
+  width: 2rem;
+  padding-top: 0.188rem;
```

Comment:
A lot of magic numbers where. Why 0.188? Why not 0.185 or so?

Reply:
<img width="448" alt="Screenshot 2021-09-12 at 12 54 04" src="https://user-images.githubusercontent.com/8509731/132983192-6988bee2-b9e3-4de2-8556-1a1ebe1f88d7.png"> 

3px ~= 0.188rem

Reply:
Then use 3px. It does not seem that there is a relation to the used primary font size.